src/Process/3dTilesProcessing.js

```diff
@@ -222,34 +222,33 @@ export function pre3dTilesUpdate(context, layer) {
 
 // Improved zoom geometry
 function computeNodeSSE(camera, node) {
+    node.distance = 0;
```

Comment:
Could you replace with this, to avoid garbage collectors and to use the methods three.js?
```js
const cameraLocalPosition = new THREE.Vector3();
const worldPosition = new THREE.Vector3();
function computeNodeSSE(camera, node) {
    node.distance = 0;
    if (node.boundingVolume.region) {
        worldPosition.setFromMatrixPosition(node.boundingVolume.region.matrixWorld);
        cameraLocalPosition.copy(camera.camera3D.position).sub(worldPosition);
        node.distance = node.boundingVolume.region.box3D.distanceToPoint(cameraLocalPosition);
    } else if (node.boundingVolume.box) {
        worldPosition.setFromMatrixPosition(node.matrixWorld);
        cameraLocalPosition.copy(camera.camera3D.position).sub(worldPosition);
        node.distance = node.boundingVolume.box.distanceToPoint(cameraLocalPosition);
    } else if (node.boundingVolume.sphere) {
        worldPosition.setFromMatrixPosition(node.matrixWorld);
        cameraLocalPosition.copy(camera.camera3D.position).sub(worldPosition);
        node.distance = node.boundingVolume.sphere.distanceToPoint(cameraLocalPosition);
    }
    if (node.distance <= 0) {
        node.distance = 0;
        return Infinity;
    }
    return camera.preSSE * (node.geometricError / node.distance);
}
```

Reply:
Done.